### ISSUE TITLE
double clear memory on MainWindow at app exit

### ISSUE THREAD

**Issue by diverlin** [Jorgen-VikingGod/Qt-Frameless-Window-DarkStyle#7]
Issue is not critical(and probably for docu), and happens on app exit, if we have a look at terminal output.
---
According to the documentation.
```
MainWindow window;
FramelessWindow framelessWindow;
framelessWindow.setContent(&window);
```

The window instance will be cleared twice, first on framelessWindow destruction(by contentLayout, on it's destruction), and second time on main exit
```
*** Error in `/media/yeti/ext_workspace/build_gcc54/cutter/cutter': double free or corruption (out): 0x00007fff1bfcc090 ***
======= Backtrace: =========
/lib/x86_64-linux-gnu/libc.so.6(+0x777e5)[0x7f528235d7e5]
/lib/x86_64-linux-gnu/libc.so.6(+0x8037a)[0x7f528236637a]
/lib/x86_64-linux-gnu/libc.so.6(cfree+0x4c)[0x7f528236a53c]
/media/yeti/ext_workspace/build_gcc54/cutter/cutter[0x406fa4]
/usr/lib/x86_64-linux-gnu/libQt5Core.so.5(_ZN14QObjectPrivate14deleteChildrenEv+0x5b)[0x7f5282efb28b]
/usr/lib/x86_64-linux-gnu/libQt5Widgets.so.5(_ZN7QWidgetD1Ev+0x358)[0x7f52837ff708]
/usr/lib/x86_64-linux-gnu/libQt5Widgets.so.5(_ZN7QWidgetD0Ev+0x9)[0x7f52837ff8d9]
/usr/lib/x86_64-linux-gnu/libQt5Core.so.5(_ZN14QObjectPrivate14deleteChildrenEv+0x5b)
```

the safer will be use 
```
MainWindow* window = new MainWindow;
FramelessWindow framelessWindow;
framelessWindow.setContent(window);
```
Maybe this should be reflected in docu? 

Thanks for a good theme!

**Comment by Jorgen-VikingGod** [comment]
thx for reporting. You are right on this - is a typical issue on most Qt examples.
I will change this today

**Comment by Jorgen-VikingGod** [comment]
My fault - I forgot, that I solved this already in August see  [commit #30a87f3](https://github.com/Jorgen-VikingGod/Qt-Frameless-Window-DarkStyle/commit/30a87f33529e3cb3cdb18417176b682907234f09)
